libretro/libretro.c

```diff
@@ -803,48 +756,38 @@ void update_variables()
         }
     }
 
-    if (AspectRatio == 1)
-        var.key = "mupen64plus-43screensize";
-    else
-        var.key = "mupen64plus-169screensize";
+    var.key = AspectRatio == 1 ? CORE_NAME "-43screensize" : CORE_NAME "-169screensize";
```

Comment:
I prefer it like this for readability
`var.key = (AspectRatio == 1 ? CORE_NAME "-43screensize" : CORE_NAME "-169screensize");`

Reply:
sure no problem.